build/scripts/build-client-docker

```diff
@@ -20,18 +20,23 @@ fi
 
 CLIENT_IMAGE_VERSION=`echo $DOCKER_CLIENT_TAG | cut -d ':' -f2`
 
-docker build --build-arg BUILD_DATE="`date --rfc-3339=seconds`" \
-             --build-arg VERSION="$CLIENT_IMAGE_VERSION" \
-             --squash \
-             --no-cache \
-             -f ${DOCKER_FILE_LOCATION}/${docker_file} \
-             -t ${DOCKER_CLIENT_TAG} \
-             ${DOCKER_FILE_LOCATION}
+docker buildx create --use --name multiarch-builder
+docker buildx inspect --bootstrap
+
+docker buildx build --build-arg BUILD_DATE="`date --rfc-3339=seconds`" \
+                    --build-arg VERSION="$CLIENT_IMAGE_VERSION" \
+                    --platform linux/amd64,linux/arm64 \
+                    --squash \
+                    --no-cache \
+                    --push \
+                    -f ${DOCKER_FILE_LOCATION}/${docker_file} \
+                    -t ${DOCKER_CLIENT_TAG} \
+                    ${DOCKER_FILE_LOCATION}
 
 if [ -n "${PUSH_DOCKER}" ]; then
     mkdir -p $(dirname ${docker_client_tag_file})
     echo ${DOCKER_CLIENT_TAG} > ${docker_client_tag_file}
-    docker push ${DOCKER_CLIENT_TAG}
+    docker buildx imagetools create --tag ${DOCKER_CLIENT_TAG} ${DOCKER_CLIENT_TAG}
```

Comment:
That tag is already created above, line 33. We should just push the image here, I believe we just need to revert this line.